gw_eccentricity/eccDefinition.py

```diff
@@ -205,13 +208,30 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
         extra_kwargs: dict
             A dictionary of any extra kwargs to be passed. Allowed kwargs
             are:
-            spline_kwargs: dict
+            special_interp_kwargs_for_extrema: dict
+                A dictionary with a single key matching the current
+                `omega_gw_extrema_interpolation_method`. See under
+                `omega_gw_extrema_interpolation_method` for more details on
+                possible values of interpolation methods.
+
+                Example structure:
+                    {
+                        "method": {"param1": value1, "param2": value2},
+                    }
+
+                currently, the available options for "method" are: 
+
+                - "spline": default kwargs are set using
+                  `utils.get_default_spline_kwargs`
+                - "rational_fit": default kwargs are set using
+                  `utils.get_default_rational_fit_kwargs`
+
+            general_interp_kwargs: dict
                 Dictionary of arguments to be passed to the spline
                 interpolation routine
                 (scipy.interpolate.InterpolatedUnivariateSpline) used to
-                compute quantities like omega_gw_pericenters(t) and
-                omega_gw_apocenters(t).
-                Defaults are set using utils.get_default_spline_kwargs
+                interpolate data other than the omega_gw extrema.
```

Comment:
Based on this description, how about renaming special_interp_kwargs_for_extrema --> special_interp_kwargs_for_omega_gw_extrema?

Reply:
Done